src/components/WindRose/WindRoseForm.tsx

```diff
@@ -0,0 +1,94 @@
+import { format } from 'date-fns';
+import { type Dispatch, type SetStateAction, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { type ChartData } from 'react-windrose-chart';
+
+import { pollutionTypes } from '@/constants/pollution';
+import data from '@/mocks/windrose/data.json';
+
+interface WindRoseFormProps {
+  setChartData: Dispatch<SetStateAction<ChartData[]>>;
+}
+
+type WindRoseFormData = {
+  dateFrom: string;
+  dateTo: string;
+  pollution: string;
+};
+
+const DATE_FORMAT = 'yyyy-MM-dd';
+
+function WindRoseForm({ setChartData }: WindRoseFormProps) {
+  const [isError, setIsError] = useState<boolean>(false);
+
+  const { register, handleSubmit } = useForm<WindRoseFormData>({
+    defaultValues: {
+      dateFrom: format(new Date('2021-01-01'), DATE_FORMAT),
+      dateTo: format(new Date('2021-01-30'), DATE_FORMAT),
+      pollution: 'PM25'
+    }
+  });
+
+  // eslint-disable-next-line
+  const fetchData = async (formData: WindRoseFormData): Promise<void> => {
+    // @TODO replace with backend real data
+    // setChartData(null);
+    // try {
+    // const { data } = await axios.get<ChartData[]>(
+    //   `${BACKEND_URL}/windrose/data`,
+    //   {
+    //     params: formData,
+    //     paramsSerializer: { indexes: null }
+    //   }
+    // );
+    // }catch {
+    //   setIsError(true);
+    // }
+    setChartData(data);
```

Comment:
```suggestion
    setChartData(data as ChartData[]);
```
You have to cast type of data loaded from JSON to make Typescript stop complaining about wrong types